docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md

```diff
@@ -690,7 +690,40 @@ Make sure when you add to cart merchant product, it has `merchantReference` set.
 
 {% endinfo_block %}
 
-<!--### 3) Configure export to Redis and Elasticsearch-->
+### 3) Configure export to Redis
+
+1. Set up event listeners and publishers:
+
+| PLUGIN | SPECIFICATION | PREREQUISITES | NAMESPACE |
+|---|---|---|---|
+| MerchantProductWritePublisherPlugin | Finds product abstract ids for merchant products and runs product storage publisher for found product abstract ids. |   | Spryker\Zed\MerchantProductStorage\Communication\Plugin\Publisher\MerchantProductWritePublisherPlugin |
+| MerchantUpdatePublisherPlugin | Finds product abstract ids by merchant ids and runs product storage publisher with found product abstract ids |   | Spryker\Zed\MerchantProductStorage\Communication\Plugin\Publisher\Merchant\MerchantUpdatePublisherPlugin |
+
+**src/Pyz/Zed/Publisher/PublisherDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Publisher;
+
+use Spryker\Zed\Publisher\PublisherDependencyProvider as SprykerPublisherDependencyProvider;
+use Spryker\Zed\MerchantProductStorage\Communication\Plugin\Publisher\Merchant\MerchantUpdatePublisherPlugin;
+use Spryker\Zed\MerchantProductStorage\Communication\Plugin\Publisher\MerchantProduct\MerchantProductWritePublisherPlugin;
+
+class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
+{
+   /**
+     * @return \Spryker\Zed\PublisherExtension\Dependency\Plugin\PublisherPluginInterface[]
+     */
+    protected function getPublisherPlugins(): array
+    {
+        return [
+            new MerchantProductWritePublisherPlugin(),
+            new MerchantUpdatePublisherPlugin(),
+        ];
+    }
+}
```

Comment:
why in Frontend section? These plugins are already presented in this IG, please move them to the corresponding place according to https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides